openasip/src/applibs/Scheduler/Algorithms/ProgramPass.cc

```diff
@@ -72,13 +73,12 @@ void
 ProgramPass::executeProcedurePass(
     TTAProgram::Program& program, const TTAMachine::Machine& targetMachine,
     ProcedurePass& procedurePass) {
-    boost::timer totalTime;
+  auto totalTimeStart = std::chrono::steady_clock::now();
```

Comment:
Is there an indent problem here? Can you try if clang-format cleans this up correctly? 4 spaces indents.

Reply:
Fixed by hand.

I need to build a clang formatting file to match your style. clang-format has a tendency to make widespread changes to your code. (See: the rearranged headers in ControlDependanceGraph. I didn't worry about putting these back in the original order as it compiles and tests properly.)